vortex-file/src/writer.rs

```diff
@@ -317,6 +321,17 @@ impl Writer<'_> {
             .load(std::sync::atomic::Ordering::Relaxed)
     }
 
+    /// Returns the number of bytes currently buffered by the layout writers.
+    pub fn buffered_bytes(&self) -> u64 {
+        self.strategy.buffered_bytes()
+    }
+
+    /// Returns the total number of bytes (written + buffered).
+    /// This provides a more accurate estimate of the final file size.
+    pub fn total_bytes(&self) -> u64 {
```

Comment:
Remove this fn, I don't know that it reports useful _enough_ information that the caller shouldn't be forced to think about themselves.